src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
If there's anything you're going to put in this file, well and good (though I would do so now if you know *what* you're going to put here); if not, remove it rather than leaving a spurious empty file lying around.

Reply:
I will remove it